src/main/SubtitleListEditor.tsx

```diff
@@ -367,84 +374,82 @@ const SubtitleListSegment = React.memo((props: subtitleListSegmentProps) => {
   })
 
   return (
-    <HotKeys keyMap={subtitleListKeyMap} handlers={handlers}>
-      <div css={segmentStyle} style={{
-        ...props.style,
-        // Used for padding in the VariableSizeList
-        top: props.style.top !== undefined ? `${parseFloat(props.style.top.toString()) + PADDING_SIZE}px` : '0px',
-        height: props.style.height !== undefined ? `${parseFloat(props.style.height.toString()) - PADDING_SIZE}px` : '0px',
-        zIndex: '1000',
-      }}>
-
-        <textarea
-          ref={textAreaRef}
-          css={[fieldStyle, textFieldStyle]}
-          defaultValue={cue.text}
-          onKeyDown={(event: React.KeyboardEvent) => {
-            if (event.key === "Enter" && !event.shiftKey) {
-              event.preventDefault()
-              addCueBelow()
-            }
-          }}
-          onChange={updateCueText}
-          onFocus={() => setTimeToSegmentStart()}
-        />
+    <div ref={hotkeyRef} css={segmentStyle} tabIndex={-1} style={{
```

Comment:
Uhm am I see this correctly that you are using `style` which is directly translated to the `style` HTML attribute? Shouldn't this use `css`? You can simply interpolate the `segmentStyle` into that object.